include/btllib/indexlr.hpp

```diff
@@ -307,6 +386,7 @@ inline Indexlr::Indexlr(std::string seqfile,
   : seqfile(std::move(seqfile))
   , k(k)
   , w(w)
+  , q(0)
```

Comment:
If the only difference is `q(0)`, we can call the other constructor from this one to prevent repetitive code:
```c++
inline Indexlr::Indexlr(std::string seqfile,
                        const size_t k,
                        const size_t w,
                        const unsigned flags,
                        const unsigned threads,
                        const bool verbose,
                        const BloomFilter& bf1,
                        const BloomFilter& bf2) 
  : Indexlr(seqfile, k, w, 0, flags, threads, verbose, bf1, bf2)
  {}
```